app/routes/task_routes.py

```diff
@@ -1 +1,91 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response
+from ..db import db
+from .route_utilities import validate_model, create_model_from_dict
+from app.models.task import Task
+from datetime import datetime
+import os, requests
+
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix = "/tasks")
+
+@tasks_bp.post("")
+def create_task():
+    request_body = request.get_json()
+    return create_model_from_dict(Task, request_body)
+
+
+@tasks_bp.get("")
+def get_all_tasks():
+    query = db.select(Task)
+    sort_param = request.args.get("sort")
+
+    if sort_param == "asc":
+        query = query.order_by(Task.title.asc())
+    
+    if sort_param == "desc":
+        query = query.order_by(Task.title.desc())
+
+    else:
+        query = query.order_by(Task.id)
+        
+    tasks = db.session.scalars(query).all()
+
+    tasks_response = [task.to_dict() for task in tasks]
+    return tasks_response
```

Comment:
Nice work on handling the filtering logic for this route and the goal route. To D.R.Y. this up some more we could use similar logic to the`get_models_with_filters` helper function that we implemented for flasky.